.vscode/settings.json

```diff
@@ -0,0 +1,18 @@
+{
+  "eslint.autoFixOnSave": true,
+  "files.associations": {
+    "OWNERS": "json5",
+    ".eslintrc": "json",
+    ".prettierrc": "json",
+    ".renovaterc.json": "json",
+    ".vscode/settings.json": "json",
+    ".codecov.yml": "yaml",
+    ".lando.yml": "yaml",
+    ".lgtm.yml": "yaml",
+    ".travis.yml": "yaml"
```

Comment:
Do we need to explicitly define file associations for files with appropriate filetype suffixes?

Reply:
I did so because VS Code sometimes auto-detects filetypes based on filename and sometimes based on which editor extensions are installed. For example. `.eslintrc` (which is json) is not detected as such, while `.prettierrc` is. Also, there is some special-casing for dotfiles.

<img width="385" alt="" src="https://user-images.githubusercontent.com/26553114/67110396-a5c93880-f1a0-11e9-856b-5079604bdc99.png">

I have removed `.vscode/settings.json` from the list since it is actually a jsonc file and non json, and the override has no effect.

Reply:
Cool, thanks for the explanation!